src/ui/text.ts

```diff
@@ -238,10 +217,21 @@ export function getTextStyleOptions(
   return { scale, styleOptions, shadowColor, shadowXpos, shadowYpos };
 }
 
-export function getBBCodeFrag(content: string, textStyle: TextStyle, closeFragment: boolean = false): string {
-  const uiTheme = settings.display.uiTheme ?? UiTheme.DEFAULT;
-  const openingFragment = `[color=${getTextColor(textStyle, false, uiTheme)}][shadow=${getTextColor(textStyle, true, uiTheme)}]`;
-  const closingFragment = closeFragment ? "[/color][/shadow]" : "";
+export function getBBCodeFrag(
```

Comment:
NIT: `Frag` is a weird shortening here. Sounds like a grenade.


```suggestion
export function getBBCodeFragment(
```